src/hotel_reservation/paths.py

```diff
@@ -0,0 +1,46 @@
+import os
+from pathlib import Path
+
+import yaml
+from dotenv import load_dotenv
+
+
+class AllPaths:
+    def __init__(self) -> None:
+        load_dotenv()  # Load environment variables from .env file if present
+
+        if self.is_databricks():
+            # Running in Databricks
+            self.filename_config = Path(
+                os.environ.get(
+                    "DATABRICKS_CONFIG_PATH",
+                    "/Workspace/Users/cremerfederico29@gmail.com/marvelmlops-cremerf/project-config.yml",
```

Comment:
This should be dynamic. You will see next week, keep it for now

Reply:
took note to remind it later.